### PR TITLE
Update Chinese translation (2022.8.27)

### PR DESCRIPTION
**Pull request by Bill-Haku**
Translation includes almost all the new vocabularies. (I didn't translated them one by one from the Git commit history, as there's too many commits. I looked through all the files and added translations. BTW, do you have any suggestion on finding out your updates effectively without any missing?)

### CODE REVIEW

**Comment by xicri** [comment]
@Bill-Haku Thanks, I'll take a look!

> BTW, do you have any suggestion on finding out your updates effectively without any missing?

I was considering to create a page to list words without Chinese translation.

Until I create such a page, it may help you to write a script to find words without `zhCN`.
It may be easier to analyze compiled JSON rather than analysing source JSON5s. Here's the JSON file:
https://dataset.genshin-dictionary.com/words.json

**Comment by Bill-Haku** [comment]
@xicri thanks. That will help me a lot.

File: dataset/dictionary/characters.json5 [hunk 1]
```diff
@@ -1173,6 +1176,7 @@
     en: "Khedive Pursina",
     ja: "ヘディーヴ・プルシナ",
     // zhCN: "海迪夫", // TODO zh-translation for Pursina
+    // PS: It seems that official translation of Pursina is not provided in the game.
```
Comment (xicri) [inline dataset/dictionary/characters.json5:1179]: Maybe this?
Khedive Pursina's Chinese translation

File: dataset/dictionary/locations.json5 [hunk 2]
```diff
@@ -1112,90 +1122,105 @@
...
     en: "Mawtiyima Forest",
     ja: "マウティーマ稠林",
+    zhCN: "无郁稠林",
     pronunciationJa: "マウティーマちゅうりん",
     tags: [ "sumeru", "location" ],
   },
   {
     en: "The Palace of Alcazarzaray",
     ja: "アルカサルザライパレス",
+    zhCN: "",
```
Comment (xicri) [inline dataset/dictionary/locations.json5:1154]: Maybe you forgot to add this?

File: dataset/dictionary/story.json5 [hunk 3]
```diff
@@ -323,55 +323,62 @@
...
     ja: "ダリオッシュ",
+    zhCN: "游学者", // TODO: Need check
     tags: [ "sumeru", "title" ],
     notes: "遊学者を意味する言葉。スメール教令院の素論派に多いとされる",
   },
   {
     en: "Haravatat",
     ja: "知論派 / ハルヴァタット学院",
-    // zhCN: "知论派", // TODO zh-translation for ハルヴァタット学院
+    zhCN: "知论派 / 室罗婆耽学院", //
```
Comment (xicri) [inline dataset/dictionary/story.json5:367]: Can you delete the comment?

```suggestion
    zhCN: "知论派 / 室罗婆耽学院",
```

File: dataset/dictionary/locations.json5 [hunk 4]
```diff
@@ -1112,90 +1122,105 @@
...
     tags: [ "sumeru", "location", "facility" ],
   },
   {
     en: "Chatrakam Cave",
     ja: "チャトラカム洞窟",
+    zhCN: "卡萨扎莱宫",
```
Comment (xicri) [inline dataset/dictionary/locations.json5:1160]: Maybe this is the translation for The Palace of Alcazarzaray.

**Review by xicri — CHANGES_REQUESTED** [review]

File: dataset/dictionary/locations.json5 (hunk 4 above)
Comment (Bill-Haku) [reply dataset/dictionary/locations.json5:1160]: I'm sorry, it's misplaced here.

File: dataset/dictionary/characters.json5 [hunk 5]
```diff
@@ -1175,8 +1175,7 @@
   {
     en: "Khedive Pursina",
     ja: "ヘディーヴ・プルシナ",
-    // zhCN: "海迪夫", // TODO zh-translation for Pursina
-    // PS: It seems that official translation of Pursina is not provided in the game.
+    // zhCN: "海迪夫·普尔希纳",
```
Comment (xicri) [inline dataset/dictionary/characters.json5:1178]: You forgot to uncomment this.

**Review by xicri — CHANGES_REQUESTED** [review]

**Review by xicri — APPROVED** [review]
Thanks, looks good!

**Pull request merged**